php/Terminus/Helpers/Input.php

```diff
@@ -259,7 +259,11 @@ public static function yesno($question, $params = array()) {
 
     $answer = trim(fgets(STDIN));
 
-    $is_no = ($answer != 'y');
-    return $is_no;
+    if ($answer == 'y') {
+      return TRUE;
+    }
+    else {
```

Comment:
also, I think our styleguide says these should go on the same line. e.g. `} else {`